lib/io_adapters.py

```diff
@@ -1,75 +1,49 @@
 import tensorflow as tf
 from tensorflow import keras
 from tensorflow.keras import layers
-
 import numpy as np
 
-
-
 def create_input_adapter(input_shape, size=16, depth=40, activation=None):
   """Creates an input adapter module for the input image.
-  The input adapter transforms input image of given shape 
+  The input adapter transforms input image of given shape
   into a tensor of target shape.
 
   Arguments: 
-    input_shape: shape of input image (HxWxC). Image width and height 
-      must be devidible by size. H*W*C must be less than or equal
-      to size*size*depth.
-
-    size: height and width of the output tensor after space2depth operation. 
-
+    input_shape: shape of input image (HxWxC). Image width and height
+    must be devidible by size. H and W must be greater than or equal to `size`.
+    size: height and width of the output tensor after space2depth operation.
     depth: number of channels in the output tensor.
+    activation: conv layer activation function."""
+  h, w, c = input_shape
+  if h < size or w < size:
+    raise ValueError('Input height and width should be greater than `size`.')
+  # `block_size` of space2depth
+  block_size = min(h / size, w / size)
+  if depth % (block_size * block_size) != 0:
+    raise ValueError('depth value is not devisible by the computed block size')
 
-    activation: conv layer activation function"""
-  def _get_parameters():
-    h, w, c = input_shape
-    if h % size != 0 or  w % size != 0:
-      raise ValueError(
-          'input height and width should be devisible by output size.')
-    num_out_pixels = size * size * depth
-    num_in_pixels = h * w * c
-    if num_in_pixels >= num_out_pixels:
-      raise ValueError(
-          'input H*W*C should not be smaller than output H*W*C.')
-
-    # `block_size` of space2depth
-    block_size = min(h / size, w / size)
-    if depth % (block_size * block_size) != 0:
-      raise ValueError('depth value is not devivible by '
-      'the computed block size')   
-    # number of 1x1 conv filters 
-    num_conv_filters = depth / (block_size*block_size)
-    return num_conv_filters, block_size
-
-  num_filters, block_size = _get_parameters()
   # creating an adapter model
   inputs = keras.Input(shape=input_shape)
-  x = layers.Conv2D(num_filters, 1, activation=activation)(inputs)
-  outputs = tf.nn.space_to_depth(x, block_size)
+  s2d = tf.nn.space_to_depth(inputs, block_size)
+  outputs = layers.Conv2D(filters=depth,
+                          kernel_size=1, activation=activation)(s2d)
   model = keras.Model(inputs, outputs, name='in_adapter')
   return model
 
-
-
-
 def create_output_adapter(input_shape, block_size=None, pool_stride=None,
-                        activation='swish', depthwise=True):
-  """ Creates an output adapter module that processes tensors before 
+                          activation='swish', depthwise=True):
+  """ Creates an output adapter module that processes tensors before
```

Comment:
Whenever possible, each docstring should start with a single line summary, then an empty line, then (optionally) a longer description, then (optionally) arguments section and then (again optionally) "Returns" and "Raises" sections. Try doing this for all functions in all files.